src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -564,6 +564,7 @@ void createCompoundJustification() {
 
             markJustification(personTypeAssertion, textJustification);
             markJustification(putin, textJustification);
+            addSourceDocumentToJustification(textJustification, "NYT_PARENT_ENG_20181231_03");
```

Comment:
I wonder if it would be more "interesting" to use this new API in an example with two documents, such as relationWhereBothEndpointsAreAmbiguousCrossDocEntitiesVersion() or relationWhereBothEndpointsAreAmbiguousNISTRestrictedVersion().  I guess since these are just the unrestricted AIF examples, we'd want to find a case where there's some utility to adding a sourceDocument, like where there are some document elements that belong to more than one document.  Anyway, we don't absolutely need a change here, but I thought we might be able to find a more interesting example to which to add the sourceDocument.  LMK what you think.

Reply:
I agree that it would be more interesting, but I'm not sure it would be any more helpful. NIST requires that all justifications have this extra property and this shows the researchers how to use it. In fact, all the tests with justifications that use the nist validator will have to change when that restriction is implemented